cpp/ql/lib/semmle/code/cpp/models/interfaces/DataFlow.qll

```diff
@@ -18,7 +19,7 @@
  * Note that this does not include partial copying of values or partial writes
  * to destinations; that is covered by `TaintModel.qll`.
  */
-abstract class DataFlowFunction extends Function {
+abstract class DataFlowFunction extends PartialFlowFunction {
```

Comment:
This is a FP since [StrDupPtr](https://github.com/github/codeql/blob/be9759fbad706c784e7fc2a8b0e8a977ec23de37/cpp/ql/test/library-tests/dataflow/dataflow-tests/TestBase.qll#L68-L68) is only in test files.